### DISCUSSION TITLE
Help with Netmiko HP procurve enable issue

### DISCUSSION THREAD

**Discussion by stinwar** [ktbyers/netmiko#2825] [Q&A]
Hi Kirk,

Reaching out to you to seek some help with Netmiko and HP procurve enable issue. I see that you have committed some changes related to same https://github.com/ktbyers/netmiko/commit/77718066617f9e545c9baf56593ccb31b947222a into certain branch.

Could you help with how can I override just this file from netmiko module? I have installed netmiko into my RHEL box via pip. I do not want to change global file as I am using this for other comware devices also.

Any help here is appreciated.

Thanks,
Shyam

**Comment by ktbyers** [comment]
@stinwar Can you provide a bit more details regarding what the issue you are running into is?

**Reply by stinwar** [reply]
@ktbyers I am using netmiko library to pull/push configuration from HP procurve devices. I am having difficulty getting into enable mode as it send "enable" as user name. I saw that you have done fix related to that but looks like not merged into master or released into production.

I was wondering, only for this automation I would override hp_procurve.py file with your updates without modify global pip netmiko package for all users. Hence I thought is there a way to override just HP procurve?

import hp_procurve.py [local file in project folder]
netmiko.modules["hp_procurve"] = __import__('hp_procurve') [trying to override netmiko hp_procurve module with local fix only for this project

Hope this explains the problem statement. Keen to hear from you.

**Reply by ktbyers** [reply]
I will probably release a new version of Netmiko in about a week (hopefully).

Probably the best solution is just to wait for that to be released.

 I think trying to reimplement a change on a single file would probably be riskier than just using the Netmiko develop branch (which sounds like it might not be the good idea since you are sharing it among multiple users).

**Reply by stinwar** [reply]
@ktbyers , were you able to merge this and release new version of code?

**Reply by ktbyers** [reply]
Yes, Netmiko 4.1.1 was released o pypi a  little while ago.

https://pypi.org/project/netmiko/#history

**Reply by stinwar** [reply]
Thanks @ktbyers , it worked flawlessly.
